README.md

```diff
@@ -1,65 +1,118 @@
-### How to build: (run from root dir)
+<img height="90" alt="MongoDB Atlas JDBC Driver" align="right" src="resources/media/MongoDBAtlasJDBC.png" />
+
+# MongoDB Atlas JDBC Driver
+
+MongoDB Atlas JDBC provides SQL connectivity to [Atlas](https://www.mongodb.com/atlas) databases and datalakes for client applications developed in Java.
+MongoDB Atlas JDBC is a JDBC Type 4 driver compatible with the JDBC 4.2 specification.
+
+### Prerequisites
+MongoDB Atlas JDBC driver requires Java 1.8 or higher.
+
+## Getting the Driver
+
+### Download and Install
+You can download the precompiled driver (jar) from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).
+
+### Build From Source
```

Comment:
suggestion: let's group all these build instructions under a "Contributing" or "development" heading alongside the integration testing instructions below

Reply:
Yes, good idea. Done!